packages/workbox-expiration/Plugin.mjs

```diff
@@ -155,7 +163,7 @@ class Plugin {
       return true;
     }
 
-    // If we have a valid headerTime, then our response is fresh iff the
+    // If we have a valid headerTime, then our response is fresh if the
```

Comment:
https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Ahh. I just saw the red underline in my editor and changed it without thinking.

Reply:
Good catch. Updated.